users/pvinis/rules.mk

```
@@ -0,0 +1,15 @@
+# add userspace file
+SRC += pvinis.c
+
+AUDIO_ENABLE     = no  # piezo speaker sounds
+RGBLIGHT_ENABLE  = no  # rgb leds underlight
+TAP_DANCE_ENABLE = yes
+BACKLIGHT_ENABLE = no  # leds under keycaps
+#MOUSEKEY_ENABLE  = no
+#SLEEP_LED_ENABLE = no  # no led blinking while sleeping
+#NKRO_ENABLE      = yes
+
+# make firmware smaller
+EXTRAFLAGS += -flto
```

This should be replaced, actually:
```suggestion
LINK_TIME_OPTIMIZATION_ENABLE = yes
```

This also disables "ACTION_MACRO" and "ACTION_FUNCTION" features, which are depreciated.